draft-ietf-pquip-pqc-engineers.md

```diff
@@ -774,7 +774,9 @@ Numerous commercial solutions are available for both detecting hard-coded crypto
 
-Hybrid key exchange increases computational and bandwidth costs by performing both a classical and a post-quantum key exchange in parallel. However, it provides security redundancy against potential weaknesses in PQ algorithms and allows for a gradual transition to PQC without breaking compatibility with existing systems. While hybrid key exchange introduces some additional overhead, the impact of traditional key exchange algorithms is quite small (e.g., key size), making the overall increase in resource usage manageable for most systems. In highly constrained environments, however, hybrid key exchange may be impractical due to its higher resource requirements compared to either pure PQ or traditional key exchange.
+Hybrid key exchange performs both a classical and a post-quantum key exchange in parallel. It provides security redundancy against potential weaknesses in PQ algorithms, allows for a gradual transition of trust in PQC algorithms, and, in backward-compatible designs, enables gradual adoption without breaking compatibility with existing systems. For instance, in TLS 1.3, a hybrid key exchange can combine a widely supported classical algorithm, such as X25519, with a post-quantum algorithm like ML-KEM. This allows legacy clients to continue using the classical algorithm while enabling upgraded clients to proceed with hybrid key exchange. In contrast, overhead-spreading hybrid designs focus on reducing the PQ overhead. For example, approaches like those described in {{?I-D.hale-mls-combiner}} amortize PQ costs by selectively applying PQ updates in key exchange processes, allowing systems to balance security and efficiency. This strategy ensures a post-quantum secure channel while keeping the overhead manageable, making it particularly suitable for constrained environments.
+
+While some hybrid key exchange options introduce additional computational and bandwidth overhead, the impact of traditional key exchange algorithms (e.g., key size) is typically small, helping to keep the overall increase in resource usage manageable for most systems. In highly constrained environments, however, hybrid key exchange may be impractical due to its higher resource requirements compared to pure post-quantum or traditional key exchange approaches. However, some hybrid key exchange designs distribute the PQC overhead, making them more suitable for constrained environments. The choice of hybrid key exchange design depends on the specific system requirements and use case, so the appropriate approach may vary.
```

Comment:
Suggested slight change for clarity:
"environments, however, those hybrid key exchange protocols may be impractical due to their higher"

Reply:
Thanks, fixed.